views/home.ejs

```diff
@@ -1 +1,32 @@
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In eu lacus non mauris condimentum congue vel quis odio. Sed lobortis velit eu eros posuere, vel viverra elit aliquam. Donec porttitor odio sit amet lorem bibendum cursus. Phasellus velit felis, sodales vitae ante lobortis, dictum feugiat velit. Ut convallis lorem ac quam ultricies faucibus. Nam viverra rhoncus dolor, id porta tortor pellentesque et. Duis elementum, magna nec pretium auctor, leo augue rutrum arcu, id pharetra leo est quis odio.</p>
+              <button onclick="window.location.href='/about'" type="button" class="section-2btn">LADC HISTORY</button>
+          </div>
+          <div class="col-lg">
+              <h3 class="section-2headings">ANNOUNCEMENTS</h3>
+              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In eu lacus non mauris condimentum congue vel quis odio. Sed lobortis velit eu eros posuere, vel viverra elit aliquam. Donec porttitor odio sit amet lorem bibendum cursus. Phasellus velit felis, sodales vitae ante lobortis, dictum feugiat velit. Ut convallis lorem ac quam ultricies faucibus. Nam viverra rhoncus dolor, id porta tortor pellentesque et. Duis elementum, magna nec pretium auctor, leo augue rutrum arcu, id pharetra leo est quis odio. </p>
+               <hr>
+               <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. In eu lacus non mauris condimentum congue vel quis odio. Sed lobortis velit eu eros posuere, vel viverra elit aliquam. Donec porttitor odio sit amet lorem bibendum cursus. Phasellus velit felis, sodales vitae ante lobortis, dictum feugiat velit. Ut convallis lorem ac quam ultricies faucibus. Nam viverra rhoncus dolor, id porta tortor pellentesque et. Duis elementum, magna nec pretium auctor, leo augue rutrum arcu, id pharetra leo est quis odio.</p>
+               <button  type="button" class="section-2btn">LEARN MORE</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
You have included these scripts for bootstrap only in "home.ejs" so when we move to any other page header won't be able to access these so add these in layout.ejs.

Reply:
sure sir

Reply:
you would have to move     _<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">_
too. After that test it locally if everything works then push and lets merge it then.

Reply:
Done sir. I have placed the cdn link at it correct position. I have tested the work locally. Thanks sir.